src/hooks/safe-apps/useRankedSafeApps.ts

```diff
@@ -1,30 +1,24 @@
-import { useMemo } from 'react'
-import { sampleSize } from 'lodash'
+import { difference } from 'lodash'
 import type { SafeAppData } from '@gnosis.pm/safe-react-gateway-sdk'
 import { getAppsUsageData, rankSafeApps } from '@/services/safe-apps/track-app-usage-count'
 
-// number of ranked Safe Apps that we include in the array
+// number of ranked Safe Apps that we want to display
 const NUMBER_OF_SAFE_APPS = 5
 
 const useRankedSafeApps = (safeApps: SafeAppData[], pinnedSafeApps: SafeAppData[]): SafeAppData[] => {
-  return useMemo(() => {
```

Comment:
The random items still keep their previous order even after removing `useMemo`.

Reply:
It doesn't even seem to be randomizing as the "random" apps are displayed in alphabetical order

Reply:
and per "Safe Apps" page order

Reply:
Oh, I accidently removed the `sampleSize` call. It should be random again now.